scripts/scil_volume_reshape.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to reshape a volume to match the resolution of another
+reference volume or to the resolution specified as in argument. The resulting
+volume will be centered in world space wrt. the reference volume or the
```

Comment:
Is this a typo? wrt?

Reply:
"with respect to"

Reply:
I agree that with the dot, it sounds weird. Should it be "in world space wrt the ref volume", maybe?

Reply:
Fixed in [30a5515](https://github.com/scilus/scilpy/pull/1019/commits/30a551512eadabb440dde90c9fed4dd0ae8db767)